tests/isTextSame.py

```diff
@@ -1,47 +1,65 @@
 #!/usr/bin/python3
 #---Dependencies---
 #sudo apt install poppler util
-
 import subprocess
+import os
 
-#Run pdftotext command in linux bash
-subprocess.call(['pdftotext', '../file.pdf', 'output'])
-
-#grab asciidoc
-adoc = open('../file.adoc', 'r')
-adocTxt = adoc.read()
-
-#get rid of + in asciidoc
-adocTxt = adocTxt.replace('+', '')
-#split string into each line
-adocLines = adocTxt.split('\n')
-    
-adocTxt = ''
-#add lines back together removing weird ones
-for line in adocLines:
-    if not line.startswith('[#'):
-        if not line.startswith('Page: link:#'):
-            adocTxt += line
-#Remove line breaks
-adocTxt = adocTxt.replace('\n', '')
-#remove spaces
-adocTxt = adocTxt.replace(' ', '')
-
-#grab output file
-pdfOutput = open('output', 'r')
-pdfTxt = pdfOutput.read()
+#pathname variables
+path_to_pdf = '../conversions/pdfs/'
+path_to_adoc = '../conversions/adocs/'
+total = 0
+passed = 0
 
-#remove line breaks
-pdfTxt = pdfTxt.replace('\n', '')
-pdfTxt = pdfTxt.replace(' ', '')
+def formatAdocTxt(adocTxt):
```

Comment:
@kgm78 
Since this script is in Python, would recommend following the official style guide for all Python code:

[PEP 8 Style Guide](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)